core/src/test/java/io/grpc/internal/BinaryLogProviderTest.java

```diff
@@ -288,48 +288,6 @@ public void onMessage(String message) {
         (int) method.parseResponse(new ByteArrayInputStream((byte[]) serializedResp.get(0))));
   }
 
-  @Test
```

Comment:
This is needed because moving the class from `io.grpc.internal` to `io.grpc` means we lose package private access to census.